usaspending_api/api_contracts/contracts/v2/agency/toptier_code/budgetary_resources.md

```diff
@@ -14,6 +14,10 @@ Returns budgetary resources and obligations for the agency and fiscal year reque
 + Response 200 (application/json)
     + Attributes
         + `toptier_code` (required, string)
+        + `agency_data_by_year` (required, array, fixed-type)
+            + (object)
```

Comment:
Usually we don't define an `object` in the response, we create a `##DataStructures` list at the bottom of the contract define it there, and then call `array[Whatever], fixed-type`.

ex: https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/reporting/agencies/overview.md